ProfileManager.js

```diff
@@ -95,14 +95,25 @@ export default class ProfileManager {
 
   async createUserDocument({
     edvClient, invocationSigner, profileAgentId, referenceId,
-    content = {zcaps: {}}
+    content = {}
   }) {
+    if(!referenceId) {
+      referenceId = edvs.getReferenceId('users');
+    }
+
     edvClient.ensureIndex({attribute: 'content.profileAgentId'});
 
+    if(!content.id) {
+      throw new TypeError('"content.id" is required.');
+    }
+
     // create the user document for the profile agent
     const userDocument = await edvClient.insert({
       doc: {
-        content: {...content, profileAgentId},
+        content: {
+          ...content,
+          zcaps: content.zcaps || {},
+        },
```

Comment:
There's a comment a bit lower that I can't tag:

```// the profile agent is only allowed to read its own doc```

In the case that we're creating a user doc for the profile, that comment doesn't make much sense. I recommend we remove it. I also feel like this function still has some code smell to it around those zcaps. What does it mean to always be delegating zcaps to some `profileAgentId` that is passed in here? I think there's still remnants of "capability set" document thought leaking in here. We'll need to deal with that with a later refactoring PR, just making note of it.

Reply:
I get the same impression elsewhere in here where we're delegating zcaps as we create EDVs, etc. to some particular profile agent. I'm hopeful that when we sort out the "on demand zcap" API (whatever that may be) we can remove all that stuff or clean it up in a way that makes it seem less awkward. As it stands it seems like we're creating a lot of "side effects" in these APIs.